src/components/ecl-button/ecl-button.stories.ts

```diff
@@ -60,41 +64,55 @@ const getArgTypes = () => {
         'flip-vertical',
       ],
     },
+    indicator: {
+      name: 'indicator',
+      type: { name: 'boolean' },
+      description: 'Show an indicator (only visible when label is hidden and icon exists)',
+      if: { arg: 'icon', truthy: true },
```

Comment:
mmh..i think this should depend on icon_only actually, this way iconOnly will appear only if an icon is selected and the indicator will appear only if the label is hidden, no..?